templates/students/attendance.html

```diff
@@ -93,6 +93,7 @@ <h3 class="text-center">Students Attendance</h3>
 
         $.ajax({
             url: "{% url 'students:ajax_attendance' %}",
+            type: 'post',
```

Comment:
`type` was used in jQuery versions prior to 1.9.0. Use `method` instead of `type` here. Also, you should write `POST` in all capital letters (as per the writing conventions).

Reply:
i have made the changes please check it out once @garg3133 .